src/monda/tests/sorad_retrieval_qc_plot_test.py

```diff
@@ -118,91 +126,73 @@ def run_example(platform_id = 'PML_SR004',
 
         if output_plots:
             log.info("Creating (ir)radiance plots")
             plots.plot_ed_ls_lt(ed, ls, lt, time, wl_output, file_id, target)
 
-
+        # Step (0) QC filters based on relative aziumuth and tilt/tilt std
+        q_az =  qc.rel_az_filter(rel_view_az, lower_azi_bound = 110, upper_azi_bound = 170)
+        q_tilt, q_tilt_std =  qc.tilt_filter(tilt_avgs, tilt_stds, upper_tilt_bound=5, upper_tilt_std_bound = np.sqrt(3))
+        q_0 =  qc.combined_filter(q_az , qc.combined_filter(q_tilt, q_tilt_std))
+   
         # Step (i) radiometric quality control filters (i.e. QC applied to l or e spectra)
         q_lt_ed = qc.qc_lt_ed_filter(ed, lt, time, wl_output, threshold = 0.020) # lt/Ed ratio (glint) filtering
         q_ed =    qc.qc_ed_filter(ed, min_ed_threshold = 500) # filters on ed and ls anomalies
         q_ls =    qc.qc_ls_filter(ls, wl_output, threshold = 1)
-        q_rad =   qc.combined_filter(qc.combined_filter(q_lt_ed, q_ed), q_ls) # combined `radiometric' qc mask
-
-        # Step (ii)  addtional qc metrics that apply to Rrs spectrum
-        q_ss =        qc.qc_SS_NIR_filter(rrswl, rrs, upperthreshold = 3, lowerthreshold = 0.5)  # similarity spectrum
-        q_maxrange =  qc.qc_rrs_maxrange(rrs, upperthreshold = 0.1, lowerthreshold = 0.00)    # filters on max and min rrs
-        q_min =       qc.qc_rrs_min(rrs, rrswl)
+        q_1 =     qc.combined_filter(qc.combined_filter(qc.combined_filter(q_lt_ed, q_ed), q_ls), q_0) # combined `radiometric' qc mask - inlcludes data passing step(0)
 
-        # (3c) algorithmic qc filters specfic to 3C (rmsd or termination at rho bounds)
+        # Step (ii): (3c) algorithmic qc filters specfic to 3C (rmsd or termination at rho bounds)
         if rrsalgorithm == '3c':
             rmsd_3c =     np.array([response['result'][i]['c3_rmsd'] for i in range(len(response['result']))]) # rmsd values used in 3C residual filter
             rho_ds =      np.array([response['result'][i]['c3_rho_ds'] for i in range(len(response['result']))]) # rho factors
             rho_dd =      np.array([response['result'][i]['c3_rho_dd'] for i in range(len(response['result']))])
             rho_s =       np.array([response['result'][i]['c3_rho_s'] for i in range(len(response['result']))])
-            q_rho =       qc.qc_3c_rho_filter(rho_ds, rho_dd, rho_s, upperbound = 0.1)
-            q_rad_resid = qc.qc_3cresidual(q_rad, rmsd_3c, tol = 1.5)
-            q_rad_3c =    qc.combined_filter(q_rho, q_rad_resid)
-
-        if rrsalgorithm == '3c':
-            q_rad_rrs = qc.combined_filter(q_rad_3c, qc.combined_filter(q_min, (qc.combined_filter(q_ss, q_maxrange)))) # recommended rrs qc mask for 3C method (combines step (i), (ii) and (iii) QC)
+            q_rho =       qc.qc_3c_rho_filter(rho_ds, rho_dd, rho_s, upperbound = 0.1) # removes data where rho terminates at optimization bounds
```

Comment:
how do we know the 3C optimisation bound?

Reply:
It will be` hard-coded' from the input parameters in the 3C processing.

Reply:
I have made a note as a in-line comment